pages/compiling/installation-reqs.rst

```diff
@@ -84,7 +84,7 @@ Conda Requirements
 
   .. code:: bash
 
-     conda create --name cantera python=3 scons cython boost numpy
+     conda create --name cantera python=3 scons cython boost numpy ruamel_yaml
```

Comment:
Technically, building Cantera no longer requires installing `ruamel_yaml` per Cantera/cantera#768. Certainly, using the conversion scripts requires `ruamel_yaml` (or `ruamel.yaml`).

Reply:
I'd leave `ruamel_yaml` in as anything else is more difficult to explain. Users should expect getting the ability to convert mechanisms except for a 'minimal installation' ... the only case I can currently anticipate would be a server install without python3 (where `ck2yaml` fails anyways).

Reply:
Compiling it may not, but running the test suite does.

Reply:
Cantera/cantera#768 sidelines `ck2yaml`, so python 2 won't be a problem?

Reply:
I was responding to @bryanwweber's comment. This bit of the instructions is targeting Python 3, anyway.

Reply:
That's true about the test suite, ok this change makes sense then.